docs/build/guides/cli/contract-lifecycle.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Contract Lifecycle
+hide_table_of_contents: true
+description: Manage the lifecycle of a Stellar smart contract using the CLI.
+---
+
+To manage the lifecycle of a Stellar smart contract using the CLI, follow these steps:
+
+1. Create an identity for Alice:
+
+```bash
+stellar keys generate alice
```

Comment:
Do you mind adding here `--no-fund` as a recommended flag to use? We are planning to make `--no-fund` to be a [default behavior in 22.0](https://github.com/stellar/stellar-cli/issues/1407) so it'd be nice if people were aware of this change (or just noting that it won't be auto-funded in future versions is fine)

Reply:
I added --no-fund and an additional step of funding. Once it becomes default we can remove the --no-fund part from this doc.